script.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function() {
+
+    /*
+		Donjeta Emerllahu
+		INFO2340
+		12/2019
+
+    */
+
+    $('#female_1_1').html;
+    var femaleName= prompt("Please enter a females firt name? (1 of 2)")
```

Comment:
So - you are collecting the needed values for all the variables. Nice!
But - you never assign the element in the HTML to have the text of those variables. Something like:
$('#female_1_1').text(femaleName);
That is why you don't see those values show up in your paragraph after all the prompt boxes.